protocol/contracts/profit/src/state/idle.rs

```diff
@@ -32,11 +32,11 @@ use crate::{
 use super::{
     buy_back::BuyBack,
     resp_delivery::{ForwardToDexEntry, ForwardToDexEntryContinue},
-    Config, ConfigManagement, ProfitCurrencies, SetupDexHandler, State, StateEnum, SwapClient,
+    Config, ConfigManagement, ProfitCurrencies, State, StateEnum, SwapClient,
 };
 
 #[derive(Serialize, Deserialize)]
-pub(super) struct Idle {
+pub(crate) struct Idle {
```

Comment:
do we need this relaxation of visibility?

Reply:
because of the error that the type is more private than the item state::State::start.

Reply:
unneeded due to the way we start a State